### PR TITLE
Functional tests for metrics micro service

### PR DESCRIPTION
**Pull request by ehenneken**

### CODE REVIEW

File: v1_0/api/metrics.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        # Test getting metrics for just one bibcode via GET
+        r = authenticated_user.get('/metrics/%s' % bibcodes[0])
+        # We should get a 200
+        self.assertEqual(r.status_code, 200)
+        # And posting just one bibcode should work too
+        r = authenticated_user.post('/metrics', json={'bibcodes': bibcodes[:1]})
+        # We should get a 200 status
+        self.assertEqual(r.status_code, 200)
+    
+    def test_bumblebee_user(self):
```
Comment (romanchyla) [inline v1_0/api/metrics.py]: @ehenneken, hi Edwin,
do you think the authenticated user and the bumblebee user will diverge in the future? (in terms of access rights; access to different functionality as opposed to just having different max-limits)

if not, you could consider using one test for both, something along the lines of:

```
def test_authenticated_user(self, user=authenticated_user):
   r = user.post('metrics', json={'bibcodes': bibcodes})
   ....

def test_bumblebee_user(self)
    test_autenticated_user(user=bumblebee_user)
```

Comment (ehenneken) [inline v1_0/api/metrics.py]: Hi Roman. At the moment I don't see a reason for divergence, but I'd like
to leave that possibility open. Is there a compelling reason to have just
one test?

-- 
=============================================================
Edwin Henneken                            ehenneken@cfa.harvard.edu
NASA Astrophysics Data System             IT Specialist
Harvard - Smithsonian                              http://
<http://adslabs.org>adslabs.org
 Center for Astrophysics                           http://ads.harvard.edu
60 Garden St. MS 83, Cambridge, MA 02138           Room P-129
                           ORCID 0000-0003-4264-2450
=============================================================

Comment (ehenneken) [inline v1_0/api/metrics.py]: Roman. After a bit more thought I decided to follow your suggestion. I'll
update my module.

-- 
=============================================================
Edwin Henneken                            ehenneken@cfa.harvard.edu
NASA Astrophysics Data System             IT Specialist
Harvard - Smithsonian                              http://
<http://adslabs.org>adslabs.org
 Center for Astrophysics                           http://ads.harvard.edu
60 Garden St. MS 83, Cambridge, MA 02138           Room P-129
                           ORCID 0000-0003-4264-2450
=============================================================

**Comment by jonnybazookatone** [comment]
Just to double check @romanchyla, any problem if I merge this?

**Pull request merged**